playbooks/group_vars/all/global.yml

```diff
@@ -1,108 +1,30 @@
 ---
 # run roles based on certain params
+skip_clean: false
 skip_prereqs: false
-force_minishift_install: false
-setup_minishift: true
-start_minishift: true
-force_repo_clone: true
-setup_jenkins: true
-setup_fedmsg: true
-setup_containers: true
+skip_artifacts: false
+skip_minishift: false
 
-# default location for minishift
-minishift_dest_dir: "{{ ansible_env.HOME }}/minishift"
+# libvirt group used in roles/prereqs/tasks/install_kvm_plugin.yml
+remote_user: "{{ ansible_user | default(lookup('env', 'USER')) }}"
 
-# default minishift profile name
-profile: minishift
+# kvm plugin url
+kvm_plugin_url: https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-centos7
 
-# disk and memory size
-disk_size: 40gb
-memory: 6400mb
+# default location for minishift & artifacts
+minishift_dest_dir: "{{ ansible_env.HOME }}/minishift"
 
-# base device size
-# for rootfs in pods
-basedevice_size: 20G
+# minishift url and release
+github_token: '62e4ec8af436dc15c3ce' 
```

Comment:
Token should not be set by default and I doubt this token works. Github token is just optional workaround for rate limiting. And BTW there is a different PR (#11) already for it :-)

Reply:
I would like this as default. If the user have the same IP will continue working on this token. If the token are wrong please inform the right token, please. About the PR #11 we continue in this work force as mentioned.

Reply:
@firemanxbr all tokens are personal and tightly coupled with some account. Feel free to add your token there :-). But it is not a good idea from security POV.

> About the PR #11 we continue in this work force as mentioned.

I do not mind for this time but that's not a standard github workflow. I hope it will be done properly next time :-)